### DISCUSSION TITLE
"all checks have failed" error message when requesting pull

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/home#19] [Q&A] [answered]
i got the error message below when i made PRs for both my vocabulary.md (https://github.com/lab-antwerp-1/home/pull/15) and when trying to merge my branch in my group2 repo (https://github.com/lab-antwerp-1/group-2/pull/4). 
i noticed this has happened to others as well. 

any advice on how to fix this issue will be greatly appreciated. thank you!

<img width="1166" alt="Screen Shot 2021-10-06 at 1 12 10 PM" src="https://user-images.githubusercontent.com/89319835/136192370-1a4b38eb-c13b-4f17-b6cd-d483189927b4.png">

**Comment by colevandersWands** [comment, marked as answer]
@aviv82 , have you read through the **Code Quality Checks** section of the repo README?  You should be able to run all of these scripts locally to find and fix the mistakes _before_ pushing.  Then when you open the PR the checks should pass

**Comment by colevandersWands** [comment]
@aviv82 , update.  turns out there was a mistake in the action (my bad). if you copy-paste [this updated version](https://github.com/HackYourFutureBelgium/template-markdown/blob/master/.github/workflows/lint.yml) into your group repo it should be good.  you can just paste it into `main`, no need for a PR

**Reply by aviv82** [reply]
thanks @colevandersWands !
